lib/ansible/module_utils/network/asa/asa.py

```diff
@@ -52,14 +52,7 @@
 }
 
 asa_top_spec = {
-    'host': dict(removed_in_version=2.9),
-    'port': dict(removed_in_version=2.9, type='int'),
-    'username': dict(removed_in_version=2.9),
-    'password': dict(removed_in_version=2.9, no_log=True),
-    'ssh_keyfile': dict(removed_in_version=2.9, type='path'),
     'authorize': dict(type='bool'),
```

Comment:
So, this wasn't marked to be removed, but I think we want to.

Reply:
Yup it seems this option was missed out. We can remove it since `authorize` is also part of `provider`